src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,13 @@
 import React from 'react';
+import { User } from '../../types/User';
+import './UserInfo.scss';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+interface UserInfoProbs {
+  user: User,
+}
+
+export const UserInfo: React.FC<UserInfoProbs> = (props) => (
+  <a className="UserInfo" href={`mailto:${props.user?.email}`}>
+    {props.user?.name}
```

Comment:
They are not necessary since you are sure you have the user
```suggestion
  <a className="UserInfo" href={`mailto:${props.user.email}`}>
    {props.user.name}
```